DESCRIPTION

```diff
@@ -0,0 +1,55 @@
+Type: Package
+Package: GAMBLR.viz
+Title: Make Plots for Genomic Analysis of Mature B-cell Lymphomas in R
+Version: 0.1.0
+Authors@R:
+    person("Vladimir", "Souza", , "vsouza@bcgsc.ca", role = c("aut", "cre"))
+Description: This package is part of the Genomic Analysis of Mature B-cell
+    Lymphomas (GAMBL) project developed by the Morin Lab.
+License: MIT + file LICENSE
+Depends:
+    R (>= 4.1)
+Imports: 
+    broom,
+    circlize,
+    ComplexHeatmap,
+    cowplot,
+    data.table,
+    dplyr,
+    forcats,
+    g3viz,
+    GAMBLR,
+    GAMBLR.helpers,
+    ggbeeswarm,
+    ggplot2,
+    ggpubr,
+    ggrepel,
+    ggsci,
+    ggthemes,
+    graphics,
+    grDevices,
+    grid,
+    gridExtra,
+    maftools,
+    methods,
+    plotly,
+    purrr,
+    RCircos,
+    RColorBrewer,
+    readr,
+    stats,
+    stringr,
+    tibble,
+    tidyr,
+    utils,
+    wordcloud
+Suggests: 
```

Comment:
What's the purpose/reason for having the `config` package under "Suggests?

Reply:
This is to solve `devtools::check()` output
```
❯ checking dependencies in R code ... WARNING
  '::' or ':::' import not declared from: ‘config’
```
But since the dependency is not in examples, it should be in `Imports` field, instead of `Suggests`.

Reply:
Both `config` and `GAMBLR.data` were moved to `Imports`.